src/server/pachyderm_test.go

```diff
@@ -4029,6 +4032,84 @@ func TestChainedPipelinesNoDelay(t *testing.T) {
 	require.Equal(t, 2, len(jobInfos))
 }
 
+// TestMetaAlias tracks https://github.com/pachyderm/pachyderm/pull/8116
+// DAG:
+//
+// A
+// |
+// B
+// |
+// C
+func TestMetaAlias(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
```

Comment:
Just a code style thing - we should generally avoid newlines inside of functions. If it feels necessary, it probably means the space should be filled with a comment